vpn/dialer.go

```diff
@@ -96,32 +117,29 @@ func (td TunDialer) DialTimeout(network, address string, timeout time.Duration)
 
 // DialContext connects to the address on the named network using
 // the provided context.
```

Comment:
Please document how the underlying `tun` is created _just once_ and how this function is basically the equivalent of a Java `synchronized` method in that only one instance of it can actually run.

Reply:
> that only one instance of it can actually run

you are referring to only one instance of TunDialer per vpn.Client, correct? (since the underlying device ties one virtual tun with one vpn client).